features/admin.users.v1/components/user-profile.scss

```diff
@@ -66,3 +66,6 @@
         flex-direction: row;
     }
 }
+.link.pointing {
```

Comment:
Let's use a separate class name instead of this as this is coming from semantic ui classes

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7992/commits/070b89ffe8036eac1f8ea66388f17d6f1226f175